src/appkit.cs

```diff
@@ -12674,7 +12679,7 @@ public interface NSSlider : NSAccessibilitySlider {
 		IntPtr Constructor (CGRect frameRect);
 
 		[Export ("isVertical")]
-		nint IsVertical { get; [Mac (10, 12)] set; }
+		nint IsVertical { get; /* [Mac (10, 12)] set; -- Radar 27222357 */ }
```

Comment:
I would add a `TODO:` or `FIXME:` so it is easier to spot, did you fill a trello card about this?